unblob/file_utils.py

```diff
@@ -234,6 +234,12 @@ def iterate_file(
         yield data
 
 
+def copy_to_file(input_file: File, output_path: Path, offset: int, length: int):
+    with output_path.open("wb") as f:
```

Comment:
I would also add a check somehow to see if the output path is within the extraction root, this is something we should check always in extractors to avoid any path traversal issue